src/GetText.Extractor/Program.cs

```diff
@@ -25,27 +26,46 @@ private static async Task<int> Main(string[] args)
             Option verbose = CommandLineOptions.Verbose;
             Option unixPathSeparator = CommandLineOptions.UseUnixPathSeparator;
             Option sort = CommandLineOptions.SortOutput;
+            Option aliasString = CommandLineOptions.GetStringAliases;
+            Option aliasDefinite = CommandLineOptions.GetParticularStringAliases;
+            Option aliasPlural = CommandLineOptions.GetPluralStringAliases;
+            Option aliasDefinitePlural = CommandLineOptions.GetParticularPluralStringAliases;
+
+            aliasString.AllowMultipleArgumentsPerToken = true;
+            aliasDefinite.AllowMultipleArgumentsPerToken = true;
+            aliasPlural.AllowMultipleArgumentsPerToken = true;
+            aliasDefinitePlural.AllowMultipleArgumentsPerToken = true;
```

Comment:
can we move these where the CommandLineOptions for aliases are statically defined:

```suggestion
internal static Option<List<string>> GetStringAliases => new Option<List<string>>(new[] { "--aliasgetstring", "-as" }, "List of aliases for GetString") { AllowMultipleArgumentsPerToken = true}; //etc for the other alias options